sdk/python/kfp/dsl/importer_node.py

```diff
@@ -25,23 +25,28 @@
 from kfp.dsl.types import type_utils
 
 URI_KEY = 'uri'
-OUTPUT_KEY = 'artifact'
 METADATA_KEY = 'metadata'
+DEFAULT_OUTPUT_KEY = 'artifact'
+DEFAULT_COMPONENT_NAME = 'importer'
 
 
 def importer(
     artifact_uri: Union[pipeline_channel.PipelineParameterChannel, str],
     artifact_class: Type[artifact_types.Artifact],
+    output_key: Optional[str] = DEFAULT_OUTPUT_KEY,
```

Comment:
Can we add tests to verify that the `output_key` is respected when exchanging data with downstream components?